### PR TITLE
Fix CI with latest CDN changes

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -7,7 +7,7 @@ GIT
 
 GIT
   remote: https://github.com/CocoaPods/Core.git
-  revision: 291a7c7e8e2ad293679baa52644f5c2f6a1b9846
+  revision: 75bc42407ded7077f2e5183ff510072059626811
```
Comment (dnkoutso) [inline Gemfile.lock:10]: will revert this, just a branch that prints more info.

**Comment by gyfelton** [comment]
@igor-makarov While the current one is pending, it has a previous build https://travis-ci.org/github/CocoaPods/CocoaPods/builds/731033040 that fails.

**Comment by gyfelton** [comment]
@igor-makarov  From the failure stacktrace in this build, I saw this line `ROOT/vendor/bundle/ruby/2.4.0/bundler/gems/Core-1c3df8f5f828/lib/cocoapods-core/cdn_source.rb:405:in `block in download_and_save_with_retries_async'` which makes me wonder any change to core recently breaks it?

**Comment by gyfelton** [comment]
additional debug stms for debugging that is in this PR: https://github.com/CocoaPods/Core/compare/fix_ci_cdn

**Pull request closed without merging**